design/cluster-api-provider-metal3/configurable_disk_cleaning.md

```diff
@@ -0,0 +1,234 @@
+<!--
+ This work is licensed under a Creative Commons Attribution 3.0
+ Unported License.
+
+ http://creativecommons.org/licenses/by/3.0/legalcode
+-->
+
+# configurable disk cleaning
```

Comment:
This title made me think we were talking about choosing individual disks. Maybe we can retitle this to something like "allow disabling disk cleaning"?

Reply:
I agree. Will change it.

Reply:
Choosing individual disks sounds interesting, but we're not quite there, +1 for renaming.

Reply:
Fixed, thanks.